website_cookie_notice/templates/website.xml

```diff
@@ -0,0 +1,45 @@
+<odoo>
+    <template
+        id="loader"
+        inherit_id="website.assets_frontend"
+        name="Cookies script loader"
+    >
+        <xpath expr=".">
+            <script
+                type="text/javascript"
+                src="/website_cookie_notice/static/src/js/accept_cookies.js"
+            />
+        </xpath>
+    </template>
+    <template id="message" name="Cookies notice">
+        <t t-set="cookie_legal_page" t-value="website.cookie_notice_legal_page_id" />
+        <div
+            t-if="request and
```

Comment:
In general, don't include logic in a template meant to be overwriten by hand.

I think it's safer to include in the main layout something like this:

```xml
<t t-if="request and not request.session.get('accepted_cookies')">
    <t t-call="website_cookie_notice.message"/>
</t>
```

And let `website_cookie_notice.message` be a simple template without any logic in it, to be overriden by end users.